Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -505,12 +505,17 @@ instance (priority := 100) ContinuousSub.measurableSub [Sub γ] [ContinuousSub 
 instance (priority := 100) ContinuousInv.measurableInv [Inv γ] [ContinuousInv γ] :
     MeasurableInv γ := ⟨continuous_inv.measurable⟩
 
+@[to_additive]
+instance (priority := 100) ContinuousConstSMul.toMeasurableConstSMul {M α} [TopologicalSpace α]
+    [MeasurableSpace α] [BorelSpace α] [SMul M α] [ContinuousConstSMul M α] :
+    MeasurableConstSMul M α where
+  measurable_const_smul _ := (continuous_const_smul _).measurable
+
 @[to_additive]
 instance (priority := 100) ContinuousSMul.measurableSMul {M α} [TopologicalSpace M]
```

Comment:
Let's rename this while we're here
```suggestion
instance (priority := 100) ContinuousSMul.toMeasurableSMul {M α} [TopologicalSpace M]
```
(it's an instance, so requires no deprecation)